plugins/kubernetes/app/models/watchers/cluster_pod_error_watcher.rb

```diff
@@ -2,34 +2,38 @@ module Watchers
   class ClusterPodErrorWatcher < BaseClusterWatcher
     include Celluloid::Notifications
 
```

Comment:
The problem was that I forgot to include the `TopicSubscription` module in this watcher. I'm not monkey patching the watchers, I'm explicitly including the module in the watchers and calling the method when needed. I just forgot to do so in this watcher and, somehow, I didn't catch it in the tests I've been running.